apps/dataset-ingestion/app/app.sh

```diff
@@ -27,17 +31,32 @@ gcloud config set auth/disable_credentials True
 
 build_coco() {
     APP="Dataset ingest (coco)"
+    python3 ../status.py --completed 0 \
+      --phases downloading \
+      --phases ingesting_images \
+      --phases ingesting_bboxes \
+      --phases ingesting_polygons \
+      --phases ingesting_pixelmaps \
+      --phases ingesting_img_pixelmap_connections \
+      --phases ingesting_images_adb_csv_clip_pytorch_embeddings_metadata \
+      --phases ingesting_images_adb_csv_clip_pytorch_embeddings_connection \
+      --phase downloading
+
     adb utils log --level INFO "${APP}: Start"
 
     date
     echo "Downloading val data..."
+
+    python3 ../status.py --completed 0
```

Comment:
it's a bit confusing what phase this is for - I noticed 'phases' versus 'phase' above, so I assume this s "downloading"?

Reply:
yes, if not specified, the values of existing keys stays same. This is an aid to shorten the command